client/src/lib.js

```diff
@@ -13,10 +13,14 @@
  * ```
  * @module
  */
-
+import { CID } from 'multiformats'
 import * as API from './lib/interface.js'
+import * as Token from './token.js'
```

Comment:
Since we are moving to different wallet based auth, I was expecting our auth `token` will go away so there will be no conflict 😅😳 ?

Reply:
we will still have tokens

Reply:
> we will still have tokens

Could those be called something else like `auth` or `claim` or someting ? Because NFT does stand for Non-fungible token, all the metadata specs refer to token so calling it something else would be more confusing.

If we can't change the auth token name, then I suggest renaming this to `NFT` but instead.

Reply:
Sync discussion we've decided to use different name for auth token.